src/events/client/messageCreate.ts

```diff
@@ -94,6 +97,43 @@ export default async function messageCreateHandler(
             return;
         }
 
+        // disable text commands if command is user facing (i.e has help documentation)
+        if (
+            invokedCommand.help &&
+            KmqConfiguration.Instance.textCommandsDisabled()
+        ) {
+            sendErrorMessage(messageContext, {
+                title: LocalizationManager.localizer.translate(
+                    message.guildID,
+                    "misc.failure.textCommandsDisabled.title"
+                ),
+                description: LocalizationManager.localizer.translate(
+                    message.guildID,
+                    "misc.failure.textCommandsDisabled.description"
+                ),
+            });
+            return;
+        }
+
+        // periodically encourage user to use slash commands instead
+        if (
+            Math.random() < 0.1 &&
+            KmqConfiguration.Instance.slashCommandRemindersEnabled()
+        ) {
+            sendInfoMessage(messageContext, {
+                title: LocalizationManager.localizer.translate(
+                    message.guildID,
+                    "misc.failure.slashCommandReminder.title"
+                ),
+                description: LocalizationManager.localizer.translate(
+                    message.guildID,
+                    "misc.failure.slashCommandReminder.description"
+                ),
+            });
+
+            await delay(1500);
```

Comment:
This delay should be removed, it will annoy the user

Reply:
The message gets flooded right away if there's no delay, and its 1 in 10 messages for a very important deprecation